src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/HologramProjector.java

```diff
@@ -89,25 +123,34 @@ private static void openEditor(Player p, Block projector) {
             return false;
         });
 
-        menu.addItem(1, new CustomItem(Material.CLOCK, "&7Offset: &e" + DoubleHandler.fixDouble(Double.valueOf(BlockStorage.getLocationInfo(projector.getLocation(), OFFSET_PARAMETER)) + 1.0D), "", "&rLeft Click: &7+0.1", "&rRight Click: &7-0.1"));
-        menu.addMenuClickHandler(1, (pl, slot, item, action) -> {
-            double offset = DoubleHandler.fixDouble(Double.valueOf(BlockStorage.getLocationInfo(projector.getLocation(), OFFSET_PARAMETER)) + (action.isRightClicked() ? -0.1F : 0.1F));
+        menu.addItem(9, new CustomItem(Material.CLOCK, "&cReset Position", "", "&eClick &7to reset hologram location"));
+        menu.addMenuClickHandler(9, (pl, slot, item, action) -> {
             ArmorStand hologram = getArmorStand(projector, true);
-            Location l = new Location(projector.getWorld(), projector.getX() + 0.5, projector.getY() + offset, projector.getZ() + 0.5);
-            hologram.teleport(l);
+            hologram.teleport(projector.getLocation().add(0.5, 0.5, 0.5));
```

Comment:
This magic value here isn't really ideal. A static constant for it would be better. Maybe change the `DEFAULT_OFFSET` to be a `double` and then just do `String.valueOf(...)` when it is needed as a String.